install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -131,6 +131,15 @@
       {{- end }}
       {{- if $.Values.global.trustDomain }}
         - --trust-domain={{ $.Values.global.trustDomain }}
+      {{- end }}
+      {{- if $.Values.global.sds.enabled }}
```

Comment:
Does this needed if controlPlaneSecurityEnabled is false? I think it should be inside the if block above.

Reply:
moved inside when controlPlaneSecurityEnabled is true